homework7/7.2/7.2/Clock.cs

```diff
@@ -0,0 +1,138 @@
+using System.Runtime.InteropServices;
+
+namespace Task7_2;
+public partial class Clock : Form
```

Comment:
```suggestion
namespace Task7_2;

public partial class Clock : Form
```
Ну и комментарии нужны